Sources/EmbraceCore/Capture/Network/URLSessionCaptureService.swift

```diff
@@ -136,13 +136,13 @@ struct SessionTaskResumeSwizzler: URLSessionSwizzler {
         if #available(iOS 15.0, tvOS 15.0, macOS 12, watchOS 8, *) {
             try swizzleInstanceMethod { originalImplementation -> BlockImplementationType in
                 return { [weak handler = self.handler] task in
-                    let captured = handler?.create(task: task) ?? true
+                    let handled = handler?.create(task: task) ?? true
 
                     // if the task wasn't captured by other swizzlers
```

Comment:
Does this comment need to be updated? It looks like it references the `!captured` that was flipped.

Reply:
Comment still makes sense

Reply:
Can we flip the language to match the updated code?

```suggestion
                    // if the task was handled by this swizzler
```